uniffi/src/testing.rs

```diff
@@ -120,17 +120,12 @@ pub fn ensure_compiled_cdylib(pkg_dir: &str) -> Result<String> {
     let cdylib_files: Vec<_> = cdylib
         .filenames
         .iter()
-        .filter(|nm| {
-            matches!(
-                nm.extension().unwrap_or_default().to_str(),
-                Some(std::env::consts::DLL_EXTENSION)
-            )
-        })
+        .filter(|nm| matches!(nm.extension(), Some(std::env::consts::DLL_EXTENSION)))
```

Comment:
Out of curiosity, were these simplifications done by clippy or similar, or something that you discovered by hand?

Reply:
Neither, these were `rustc` errors. It appears these returned `Option<Cow<str>>` before and now just `Option<&str>`